pkgs/development/python-modules/stups-berry/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, buildPythonPackage
+, fetchpatch
+, fetchPypi
+, dnspython
+, pyyaml
+, boto3
+}:
+
+buildPythonPackage rec {
+  pname = "stups-berry";
+  version = "1.0.28";
+
+  propagatedBuildInputs = [
+    dnspython
+    pyyaml
+    boto3
+  ];
+
+  patches = [
+    (fetchpatch {
+      name = "fix-pip10-compat.patch";
+      url = "https://github.com/zalando-stups/berry/commit/209587e855d93cbe31339a88ab47f1cc641dba63.patch";
+      sha256 = "0lgpz409c51d5xlw520szhb3fxrxss1kg7rdls8rwd2b95dfxp4j";
+    })
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qk15wligwzxfappajiby682np0ndvn0zvgq0b0lsn71adw82x2j";
+  };
+
+  meta = with stdenv.lib; {
+    description = "STUPS-related agent to retrieve application credentials.";
+    homepage = "https://github.com/zalando-stups/berry";
+    license = licenses.apl2;
+    maintainers = [ maintainers.mschuwalow ];
```

Comment:
please add an entry into `maintainers/maintainer-list.nix`, and add that commit before the other packages, or else it will fail to evaluate at certain points in time